src/content/docs/en/pages/products/edge-application/rules-engine.mdx

```diff
@@ -318,11 +318,9 @@ Finishes the request phase. Any behavior or other rules under this behavior won'
 <Badge>Request Phase</Badge>
 [<Badge variant="accent">Requires Application Acceleration</Badge>](/en/documentation/products/edge-application/application-acceleration/)
 
-By using the Forward Cookies behavior, you're determining that Azion forwards to the users the Set-Cookie header received from its origin, even in a cache content situation (cache hit). To prevent a user from receiving another user's `Set-Cookie` session, see the guide [How to configure cache policies for Edge Application](/en/documentation/products/guides/edge-caching/#creating-a-rule-to-bypass-cache-from-the-origin).
+By using the Forward Cookies behavior, you're determining that Azion forwards to the users the Set-Cookie header received from its origin, even when cached content is identified (HIT). 
 
-To prevent a user from receiving another user's `Set-Cookie` session, you must list all session cookies (private cookies) your application must ignore using the [Advanced Cache Key](/en/documentation/products/edge-application/cache-settings/#advanced-cache-key).
-
-> See the guide on [How to configure cache policies for Edge Application](/en/documentation/products/guides/edge-caching/#forward-cookies) for more information on Forward Cookies.
+To prevent a user from receiving another user's `Set-Cookie` session, see the guide [How to configure cache policies for Edge Application](/en/documentation/products/guides/edge-caching/#creating-a-rule-to-bypass-cache-from-the-origin).
```

Comment:
```suggestion
To prevent a user from receiving another user's `Set-Cookie` session, see the [cache policies guide](/en/documentation/products/guides/edge-caching/#creating-a-rule-to-bypass-cache-from-the-origin).
```